src/pyfaf/rpm.py

```diff
@@ -92,7 +92,8 @@ def store_rpm_deps(db, package, nogpgcheck=False):
         new.flags = r.Flags()
         evr = r.EVR()
         if evr:
-            new.epoch, new.version, new.release = rpmutils.stringToVersion(evr)
+           def evr(self):
```

Comment:
you defined this function, but you don't seem to use it. Also you removed setting of `new.epoch, new.version, new.release`.

Reply:
Also it is badly intended (there are only 3 spaces).